app/controllers/api/v2/hosts_controller.rb

```diff
@@ -202,7 +202,7 @@ def status
 * configuration
 * build
 eos
-)
+                                                                                                                                        )
```

Comment:
This is a bit odd location, perhaps break the lines in the param call so it's a bit clearer?